DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,315 @@
+
+      ```
+      set_time_format -unit ns -decimal_places 3
+      create_clock -name i_clk -period 10 [get_ports {i_clk}]
+      ```
+
+   3. Cut the clock paths for asynchronous I/O:
+      
+      ```
+      set_false_path -from [get_ports {reset_button_n}] -to * 
+      set_false_path -from [get_ports {fpga_led}] -to *
+      set_false_path -from * -to [get_ports {fpga_led}]
+      ```
+
+9. Compile the full design by clicking the 'Start Compilation' button in the Intel® Quartus® Prime GUI.
+
+      ![](README.md.assets/start-compilation-quartus.png)
+
+### Additional Documentation
+- [Explore SYCL* Through Intel® FPGA Code Samples](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of FPGAs and SYCL.
+- [FPGA Optimization Guide for Intel® oneAPI Toolkits](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) helps you understand how to target FPGAs using SYCL and Intel® oneAPI Toolkits.
+- [Intel® Arria® 10 SoC Golden System Reference Design](https://rocketboards.org/foswiki/Documentation/Arria10SoCGSRD) describes a reference design you can use with your Intel® Arria® 10 SX SoC Developer kit.
+- [Intel® Arria 10 SX SoC Development Kit](https://www.intel.com/content/www/us/en/products/details/fpga/development-kits/arria/10-sx.html) describes the Intel® Arria® 10 SX SoC Development kit in greater detail.
+- [Intel® FPGA Software Installation and Licensing](https://www.intel.com/content/www/us/en/docs/programmable/683472/current/faq.html) describes how to license Intel® Quartus® Prime Pro software.
+- [Intel® oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) helps you understand target-independent, SYCL-compliant programming using Intel® oneAPI Toolkits.
+- [Intel® Quartus® Prime Pro Edition User Guide: Getting Started](https://www.intel.com/content/www/us/en/docs/programmable/683463/current/faq.html) introduces you to the Intel® Quartus® Prime Pro software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Platform Designer](https://www.intel.com/content/www/us/en/docs/programmable/683609/current/faq.html) describes the Intel® Platform Designer software.
+- [Intel® Quartus® Prime Pro Edition User Guide: Programmer](https://www.intel.com/content/www/us/en/docs/programmable/683039/current/programmer-user-guide.html) describes the Intel® Quartus® Prime Pro Programmer software.
+
+## Running the Sample
+
+Use the `test.bat` script in the `add-quartus/system_console` directory to flash the design to your development board, and launch the system console. The included `.tcl` scripts in the `system_console` directory demonstrate how to use the System Console to interact with your IP through the JTAG Avalon Master IP on the FPGA.
```

Comment:
This is Windows specific, and it's missing the Linux instructions

Reply:
there is no linux script because I have no way to test it, and any linux user that wants to plug a board into their computer an run it can figure out the commands from the batch script.